projects/pastanaga-angular/src/styles/_checkbox.scss

```diff
@@ -53,14 +53,11 @@
 
             &:disabled {
                 background-image: url('data:image/svg+xml; utf8, <svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36"><g fill="none" fill-rule="evenodd"><polygon fill="%23' + str_slice(
-                        #{$color-check-toggle-disabled},
+                        #{$color-check-toggle},
                         2
                     ) + '" points="29.021 17.021 11.021 17.021 11.021 9.021 7.021 9.021 7.021 21.021 29.021 21.021" transform="rotate(-45 18.02 15.02)"/></g></svg>');
```

Comment:
The only difference in background image between disabled and checked was the color, so now you have the same color you should completely remove this `background-image` override